gnc/sub8_perception/nodes/marker_occ_grid.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+from __future__ import division
+
+import rospy
+import tf
+from std_msgs.msg import Header
+from geometry_msgs.msg import Pose, Point, Quaternion, Pose2D
+from nav_msgs.msg import OccupancyGrid, MapMetaData
+from sub8_msgs.srv import VisionRequest2D, VisionRequest2DResponse
+
+import cv2
+import numpy as np
+
+
+def unit_vector(vect):
+    return vect / np.linalg.norm(vect)
+
+
+def make_2D_rotation(angle):
```

Comment:
Check out [this](https://github.com/uf-mil/Sub8/blob/master/utils/sub8_ros_tools/sub8_ros_tools/geometry_helpers.py)

Don't worry about it now, but if you find yourself needing this function elsewhere, put it there